### ISSUE TITLE
Alternative installation instructions for when .flatpakref isn't associated

### ISSUE THREAD

**Issue by jonathon-love** [flathub/linux-store-frontend#288]
hi,

after following the setup instructions, users can click the "Install" button on an applications page. however, if their setup has not correctly associated .flatpakref (this happens on chromeos for example, where i don't think there's any way to associate .flatpakref with something running in crostini), they end up looking a text file.

could command line instructions get included somewhere?

perhaps under the "Install" button? ("doesn't work? use this command")

or perhaps *in* the .flatpakref? that way if someone does end up viewing that in a text editor, it will have some text explaining how to move forward.

with thanks

**Comment by bilelmoussaoui** [comment]
![image](https://user-images.githubusercontent.com/7660997/111004186-cfcc5d80-8388-11eb-8e8c-21d0362da3fc.png)
It does exists already?

**Comment by jonathon-love** [comment]
oh yup. this might need to be closer to the top, and with some explanation that this is an alternative for when the install button doesn't work.

**Comment by jimmysjolund** [comment]
Would it be possible and/or worth the effort to expand on the Quick Setup for each distro on how to install with the button and what to do if it doesn't associate correctly? Not everyone resort to command line.

**Comment by razzeee** [comment]
I'm not sure if I get what your trying to say. Your basically asking for the website to detect your distro and show graphical instructions accordingly?

I would think that clicking the install link on the apps page would lead to the install procedure on every distro anyway. Not sure there is worth to have that here again.

**Comment by jimmysjolund** [comment]
No, I'm suggesting that for instance https://flatpak.org/setup/Debian would include the instructions on what to do in case the Install button is not opening the installation as expected. It didn't on my installations in Debian, Fedora, or RHEL and there's recent question on Discourse asking on how to actually install after going through the setup as it doesn't open the right application.

I could provide instructions with/without pictures but I'm not sure I need to fork the entire project (and I'm not an angular person)?

**Comment by razzeee** [comment]
The part your linking too is a different website repo https://github.com/flatpak/flatpak.github.io

**Comment by jimmysjolund** [comment]
Ok, so Flathub doesn't provide any instructions for new users on how to actually install everything available in Flathub, but rely on the Flatpak instructions?

Different issue for sure.

**Comment by razzeee** [comment]
Well as you see, the url you linked above is on `flatpak.org` not on `flathub.org`. I don't think we want to have two places to manage these, so it's probably fine like it is.
